bookshop/src/Login.vue

```diff
@@ -0,0 +1,85 @@
+<template>
```

Comment:
You copied this file from components.. why?

Reply:
Yeah, you are right. It is my mistake. It gonna to be removed next commit,